lib/internal/errors.js

```diff
@@ -320,7 +320,7 @@ function invalidArgType(name, expected, actual) {
   if (Array.isArray(name)) {
     var names = name.map((val) => `"${val}"`).join(', ');
     msg = `The ${names} arguments ${determiner} ${oneOf(expected, 'type')}`;
-  } else if (name.includes(' argument')) {
+  } else if (name.endsWith(' argument')) {
```

Comment:
Any reason not to check the type here too?

Reply:
It's checked in the `if` part. I mean, I guess it *could* be something other than a string or an array, but it's an internal function, so I'm not worried about end users sending an object or something.

My preference here is slight, though, and I can see the point of programming extra defensively here and adding the type check. If I do that, I'll want to add a test too to cover it.

Reply:
I'm also -0 on over grounding an internal function.
IMHO if at all it should be at asserted around L309:
https://github.com/nodejs/node/blob/adbed027fb3a720ecb10f36cea19bb43cf049720/lib/internal/errors.js#L307-L309